SPECS/mariner-rpm-macros/mariner-rpm-macros.spec

```diff
@@ -87,6 +88,9 @@ mkdir -p %{buildroot}%{rcluadir}/{rpm,srpm}
 install -p -m 644 -t %{buildroot}%{rcluadir} common.lua
 install -p -m 644 -t %{buildroot}%{rcluadir}/srpm forge.lua
 
+sed -e 's|@DIST@|%{dist}|g' %{SOURCE28} > macros.dist
+install -m 644 -p macros.dist %{buildroot}/%{macrosdir}
```

Comment:
I'd move `sed` before the `mkdir -p %{buildroot}%{_rpmconfigdir}/macros.d` line and remove the new `install`, since all `macros.*` files are already installed above.

Reply:
Done